axop/plugins.py

```diff
@@ -34,6 +35,34 @@ class InternalPlugin(): # pylint: disable=too-few-public-methods
     def __init__(self, kplugin: PluginKind):
         self._kplugin = kplugin
 
+    def decode(self, input_str: str) -> str:
+        """
+        Decode base64 and unzip
+        """
+        base64_bytes = input_str.encode('utf-8')
+        message_bytes = base64.b64decode(base64_bytes)
+        uncmpstr = zlib.decompress(message_bytes)
+        message = uncmpstr.decode('utf-8')
+        return message
+
+    def is_encoded(self, input_str: str) -> bool:
```

Comment:
This is not necessary true.
is_encoded("test") will be True.

Reply:
I did not know that... interresting